.github/workflows/packaging.yml

```diff
@@ -40,6 +40,11 @@ jobs:
       DIST: ${{ matrix.platform.dist }}
 
     steps:
+      - name: Version check
+        # We need this step to run only on push with tag.
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+        uses: ./.github/actions/version_checking
```

Comment:
Well, if you use it this way, module clone and tarantool install are redundant.

Reply:
Do you mean delete steps `Clone the module` and `Setup tarantool` from `version_checking/action.yml`?

Reply:
Feel free to reorder existing CI steps